apps/frontend/src/libs/components/sidebar/sidebar-item/styles.module.css

```diff
@@ -0,0 +1,19 @@
+.sidebar-link {
```

Comment:
I don't see styles for active item. When Projects item is selected it should highlighted as on figma

Reply:
It`s a little lower, .sidebar-link:hover {